scripts/cypress.js

```diff
@@ -16,44 +16,33 @@ const argv = require('yargs')
   .option('mode', {
     describe: 'Choose a mode to run cypress',
     choices: ['run', 'open'],
-  })
-  .option('package', {
-    describe: 'Unscoped package name to load the deployed storybook for (used by PR runs only)',
-    default: 'react-components',
-    type: 'option',
-    choices: ['react-components', 'react'],
-  })
-  .option('port', {
-    describe: 'Port number storybook is running on (used by local runs only)',
-    default: 3000,
-    type: 'number',
+    default: 'open',
   })
   .demandOption('mode').argv;
 
 const isLocalRun = !process.env.DEPLOYURL;
 
 /** @type {Cypress.ConfigOptions} */
 const baseConfig = {
-  baseUrl: isLocalRun ? `http://localhost:${argv.port}` : `${process.env.DEPLOYURL}/${argv.package}/storybook`,
-  fixturesFolder: path.join(__dirname, 'cypress/fixtures'),
-  integrationFolder: '.',
-  pluginsFile: path.join(__dirname, 'cypress/plugins/index.js'),
+  video: false,
+  testFiles: path.join(process.cwd(), '**/*.e2e.tsx'),
```

Comment:
Is this going to look in node_modules as well?

Reply:
in the root node_modules ? no

Reply:
I meant in the package's node_modules